src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -2101,24 +2234,24 @@ public void putCachedCounter(ByteBuffer partitionKey, Clustering clustering, Col
         CacheService.instance.counterCache.put(CounterCacheKey.create(metadata.ksAndCFName, partitionKey, clustering, column, path), clockAndCount);
     }
 
-    public void forceMajorCompaction() throws InterruptedException, ExecutionException
+    public void forceMajorCompaction() throws ExecutionException
```

Comment:
I think this doesn't throw `ExecutionException`. Note that removing that makes the try-catch block in `CQLTester#compact()` unneeded.